x-pack/plugins/security_solution/public/resolver/store/data/selectors.ts

```diff
@@ -529,3 +529,12 @@ export const processNodePositionsAndEdgeLineSegments = createSelector(
     };
   }
 );
+
+/**
+ * Returns the `children` and `ancestors` limits for the current graph, if any.
+ *
+ * @param state {DataState} the DataState from the reducer
+ */
+export const limitsReached = (state: DataState) => {
```

Comment:
Should there be selector that returns a boolean indicating to show a message like`shouldShowTooMuchDataMessage`?

Reply:
There should be 1 place in code that defines "when children is `null`, that means there's no more data, when children is a string, that means there is more data" and there should be a test for that logic.

Reply:
https://github.com/elastic/kibana/pull/69590/commits/5c6a45349adfc523dc8c946adb7eda70fa4ea7cb